edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss

```diff
@@ -497,7 +497,7 @@ $homepage__header--gradient__color--bravo: saturate($gray, 30%) !default;
 $homepage__header--background: lighten($gray, 15%) !default;
 $homepage-background: rgb(252, 252, 252) !default;
 $course-card-height: ($baseline*18) !default;
-$course-image-height: ($baseline*8) !default;
+$course-image-height: auto;
```

Comment:
Would this affect all other images on the platform? @luismorenolopera

Reply:
yes

Reply:
Is that the desired behaviour?

Reply:
yes